contributing/adding_datasets.md

```diff
@@ -9,232 +9,109 @@ parent: Contributing to Data Commons
-
-Node: dcid:CumulativeCount_MedicalConditionIncident_COVID_19_PatientDeceased
-typeOf: dcs:StatisticalVariable
-populationType: dcs:MedicalConditionIncident
-medicalCondition: dcs:COVID_19
+populationType: dcs:HousingUnit
+occupancyStatus: dcs:OccupiedHousingUnit
+occupancyTenure: dcs:OwnerOccupied
 statType: dcs:measuredValue
-measuredProperty: dcs:cumulativeCount
-medicalStatus: dcs:PatientDeceased
+measurementDenominator: dcs:Count_HousingUnit_OccupiedHousingUnit
 ```
 
-The full set of `StatisticalVariable`s is checked into [https://github.com/datacommonsorg/data](https://github.com/datacommonsorg/data) under the appropriate [`scripts/<provenance>/<dataset>` subdirectory](https://github.com/datacommonsorg/data/tree/master/scripts/covid_tracking_project/historic_state_data).
-
-When going through the import process, see the [GitHub Development Process](https://github.com/datacommonsorg/data#github-development-process) for how to prepare for a pull request. 
-
-### Step 2: Clean the CSV.
-
-There are no restrictions on your approach for this step, only requirements for its result.
-
-1. Each `StatisticalVariable` must have its own column for its observed value.
-2. Each property in your schema must have its own column for its value, including the values of `observationAbout` and `observationDate`.
-3. Dates must be in ISO 8601 format: "YYYY-MM-DD".
-4. References to existing nodes in the graph must be `dcid`s.
-5. The cleaning script is reproducible and easy to run. Python or Golang is recommended.
+### Final touches
 
-#### Step 2a: Write the Preprocessing Script.
+#### Adding a README
 
-**Note:** Renaming the columns is not necessary. It is a style choice to maintain reference to the `StatisticalVariable` ID at all times.
-
-An example script prepending `dcid:` to the state identifiers, thus allowing later global references to the states, is available at https://github.com/datacommonsorg/data/blob/master/scripts/covid_tracking_project/historic_state_data/preprocess_csv.py.
+Follow the template at <https://github.com/datacommonsorg/data/tree/master/scripts/example_provenance/example_dataset> to construct a README for your dataset.
 
-#### Step 2b: Check In the Preprocessing Script.
+#### Final steps
 
-Check in the script and the resulting CSV to [https://github.com/datacommonsorg/data](https://github.com/datacommonsorg/data) under the appropriate [`scripts/<provenance>/<dataset>` subdirectory](https://github.com/datacommonsorg/data/tree/master/scripts/covid_tracking_project/historic_state_data).
+Check in the Template MCF file together with the cleaned CSV, its preprocessing script (if needed), and the README to [https://github.com/datacommonsorg/data](https://github.com/datacommonsorg/data) under the appropriate [`scripts/<provenance>/<dataset>` subdirectory](https://github.com/datacommonsorg/data/tree/master/scripts/fred/homeownership). If you wrote a script to automate the generation of the TMCF, you may also include that.
```

Comment:
Instead of "check in", should we say "send a pull request"? Not sure what the accepted term is for github.